src/data_provider/src/packages/pkgWrapper.h

```diff
@@ -125,25 +132,50 @@ class PKGWrapper final : public IPackageWrapper
                 }
             };
 
+            static const auto truncateVersion
+            {
+                [](const std::string & val)
+                {
+                    constexpr auto VERSION_FORMAT{R"(^([^.]+\.[^.]+\.[^.]+).*$)"};
+                    static std::regex versionFormatRegex{VERSION_FORMAT};
+                    std::string version;
+                    Utils::findRegexInString(val, version, versionFormatRegex, 1);
+                    return version;
+                }
+            };
+
             const auto getDataFnc
             {
                 [this, &filePath](std::istream & data)
                 {
                     std::string line;
 
+                    m_name = UNKNOWN_VALUE;
```

Comment:
It is incorrect to initialize the name with a space, since it is checking that this field is different from empty.

Reply:
check sysInfoMac.cpp line 186

Reply:
Mmm yes, totally. Thanks